README.md

```diff
@@ -9,6 +9,9 @@ Kannon can be installed via `pip`.
 pip install kannon
 ```
 
+# [Quick Starter](./example/quick_starter.md)
+An easy and self-contained tutorial can be found in [here](./example/quick_starter.md)!
```

Comment:
I think `example/README.md` is better because someone click example directory directly. If the file name is README.md, they can read example/README.md.

Reply:
Changed to `example/README.md`, thanks!